projects/common/src/navigation/navigation.service.ts

```diff
@@ -229,6 +231,26 @@ export class NavigationService {
     return this.findRouteConfig(path, childRoutes ? childRoutes : []);
   }
 
+  public decorateNavItem(navItem: NavItemConfig, activatedRoute: ActivatedRoute): NavItemConfig {
+    if (navItem.type !== NavItemType.Link) {
+      return { ...navItem };
+    }
+    const features = navItem.matchPaths
+      .map(path => this.getRouteConfig([path], activatedRoute))
+      .filter((maybeRoute): maybeRoute is TraceRoute => maybeRoute !== undefined)
```

Comment:
This will change to htRoute right?

Reply:
@anandtiwary after refactor, yes.

Reply:
okay

Reply:
But since this is a new addition, once your other PR is merged, we can immediately change this to HTRoute right? Or would you update it after migrating t-ui changes?

Reply:
done.